src/services/jsTyping.ts

```diff
@@ -5,21 +5,26 @@
 
 /* @internal */
 namespace ts.JsTyping {
+    interface TsdJsonCacheInfo {
+        path: string;

```

Comment:
is the `path` property used anywhere? If not, can the functions just return the JSON object instead